wasm-node/rsc/run_node.js

```diff
@@ -0,0 +1,44 @@
+Error.stackTraceLimit = 99;
+
+import init, { setup, Network, WasmNode, WasmNodeConfig, canonical_network_bootnodes, network_genesis } from "/wasm/wasm_node.js";
+
+// initialize wasm
+await init();
+// setup logging and console panic hook
+await setup();
+
+const response = await fetch('/cfg.json');
+const json = await response.json();
+
+console.log("Received config:", json);
+
+const network = json.network;
+let bootnodes = json.bootnodes
+if (bootnodes.length === 0) {
+    bootnodes = canonical_network_bootnodes(network);
+}
+const genesis = network_genesis(network);
+
+document.getElementById("network_id").value = network;
+document.getElementById("genesis").value = genesis;
+document.getElementById("bootnodes").value = bootnodes.join("\n");
+
+document.getElementById("start").addEventListener("click", async function(ev) {
+    document.getElementById("start").setAttribute("disabled", "disabled");
```

Comment:
wouldn't this do?
```suggestion
    document.getElementById("start").disabled = true;
```

Reply:
I actually think that setting .disabled to "disabled" was a bit hacky as it worked because the string "disabled" is truthy :)